benchbuild/utils/wrapping.py

```diff
@@ -25,8 +25,10 @@
 import logging
 import os
 import sys
+import typing as tp
 
 import dill
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)